do.py

```diff
@@ -276,7 +276,7 @@ def go_lint():
     )
     run([pkg])
     os.chdir("pkg")
-    run(["golangci-lint run -v"])
+    run(["golangci-lint run -v --disable staticcheck"])
```

Comment:
had to call disable cause otherwise golangci-lint shows usage of deprecation errors,

Reply:
isn't there a different approach ? we cannot disable staticcheck altogether

Reply:
included a config file for golangci-lint .golangci.yaml which ignores the static check for deprecation
the code ignored is SA1019 more info in https://staticcheck.io/docs/checks#SA1019